Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Get Golang for builder
-FROM golang:1.22.3 as builder
+FROM golang:1.22.5 as builder
```

Comment:
I'm wondering, should we have higher version of golang here (`1.22.5`) than the one defined in `go.mod` (`1.22.4`)🤔 

Maybe we should bump the go.mod version first?

Reply:
Ok, I'll do.